simra/database_test.go

```diff
@@ -64,9 +65,10 @@ func TestBolt(t *testing.T) {
 		_ = os.RemoveAll(tmpdir)
 	}()
 
-	db := OpenDB(&database.Boltdb{}, filepath.Join(tmpdir))
-	if db == nil {
-		t.Error("failed to open database")
+	db, err := OpenDB(&database.Boltdb{}, filepath.Join(tmpdir))
+	if err != nil {
+		t.Error("failed to open database. err: %s", err.Error())
```

Comment:
__[go vet]__

possible formatting directive in Error call


<!-- Comment by [SideCI](https://sideci.com) -->